code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -612,7 +612,6 @@
 	var/obj/effect/timestop/T = new /obj/effect/timestop
 	T.forceMove(get_turf(holder.my_atom))
 	T.immune += mob
-	T.timestop()
```

Comment:
Is this removing timestop from slime reactions? Won't that make sepia slimes meaningless now?

Reply:
No timestop is automatically started when the effect is made now